contracts/assemblyscript/nep21-basic/non-spec.ts

```diff
@@ -17,6 +17,12 @@ export const ERR_TOKEN_ALREADY_MINTED = 'Token has previously been minted'
 /* NON-SPEC METHODS */
 /********************/
 
+// huh!
+// so I would have to deploy this contract then immediately call this endpoint
+//
+// * could someone malicious call it in between?
+// * could we hard-code the default token owner in another way?
+// * could we make it possible to mint more in the future? is that undesirable?
 export function mint(owner_id: AccountId): void {
```

Comment:
I think several of these questions are answered by the design choices in our initial contracts like staking / delegation. let's dig into those together and see if we can improve on this hackjob of mine